base/deprecated.jl

```diff
@@ -259,3 +259,5 @@ const base64 = base64encode
 
 @deprecate sizehint(A, n) sizehint!(A, n)
 
+@deprecate randbool()      rand(Bool)
+@deprecate randbool(x...)  bitrand(x)
```

Comment:
the forwarding of `randbool` taking an `AbstractRNG` won't work as is, nor `randbool((2,3))`.